zoomx/src/main/java/com/zoomx/zoomx/networklogger/NetworkLogManager.java

```diff
@@ -11,6 +11,7 @@
 
 public class NetworkLogManager {
 
+    //TODO: check if we could do it with Rxjava
```

Comment:
Great.! 
We can add it to our backlog (features & enhancements). If you can do it you can go ahead 👍